configs/common/mozconfig

```diff
@@ -89,7 +89,6 @@ ac_add_options --enable-raw
 ac_add_options --enable-webrtc
 ac_add_options --enable-jxl
 ac_add_options --enable-av1
-ac_add_options --enable-eme=widevine
```

Comment:
DRM support varies by architecture and OS. Linux aarch64 doesn't support widevine so it can't be enabled in the common mozconfig.

Reply:
Specifically, the Linux aarch64 build will fail with 
``mozbuild.configure.options.InvalidOptionError: --enable-eme is not available in this configuration``

Tor browser also had to change theirs because of this
https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42501